components/section/index.tsx

```diff
@@ -82,7 +82,7 @@ export const Section: React.FC<PropsWithChildren<ISectionProps>> = ({
           border: 2px solid ${borderColor};
           border-radius: 2px;
           position: relative;
-          margin: 10px 0 10px;
+          margin: 20px 0 40px;
```

Comment:
This can have big consequences, especially on the print part of things. Are you positive it does not breaks UI ?

Reply:
Check with print and the layout is OK.
It fixes the UI : 

**BEFORE**
![image](https://github.com/etalab/annuaire-entreprises-site/assets/1775934/9e47a1d1-d554-4f18-9a6c-05e940c91c61)


**AFTER** 
![image](https://github.com/etalab/annuaire-entreprises-site/assets/1775934/2615837c-14bb-4c75-9b4f-2b0d00a50081)

Reply:
(it does reduce the spacing when using separator, but it's for more coherence overhall)